keyboards/ztboards/after/info.json

```
@@ -1,12 +1,12 @@
 {
     "keyboard_name": "ZTBoards After", 
-    "url": "ztboards.myshopify.com", 
-    "maintainer": "phlopsuh", 
+    "url": "ztb.mx", 
+    "maintainer": "ZTBoards phlop", 
```

The value here is intended to be the maintainer's GitHub name.

```suggestion
    "maintainer": "phlopsuh", 
```